app/src/main/res/raw/control_files.json

```diff
@@ -111,5 +111,12 @@
   "chargeOn": "1",
   "chargeOff": "0",
   "experimental": true
+},{
+  "file": "/sys/class/power_supply/chargalg/disable_charging",
+  "label": "chargalg/disable_charging",
+  "details": "Sony XPERIA Mini ST15i",
+  "chargeOn": "1",
```

Comment:
Aren't "disable_charging" and "chargeOn: 1" contradictory? Your screenshots prove so, too. You `echo 1` to disable charging (chargeOff) and `echo 0` to enable it (chargeOn).